app/frontend/src/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -141,9 +153,25 @@ export default function HostRequestSendField({
                 HostRequestStatus.HOST_REQUEST_STATUS_ACCEPTED ||
               hostRequest.status ===
                 HostRequestStatus.HOST_REQUEST_STATUS_CONFIRMED) && (
-              <FieldButton callback={handleReject} isLoading={isButtonLoading}>
-                Reject
-              </FieldButton>
+              <ConfirmationDialogWrapper
+                title="Are you done messaging?"
+                message={
+                  `You can only message users you have added as your friends. ` +
+                  `Please make sure you are done chatting before you  ` +
+                  `reject their request.`
```

Comment:
We use `constants.ts` files for strings - so you'd move these two strings there and export them as `CLOSE_REQUEST_DIALOG_TITLE` or something, and use those here